### PR TITLE
[DNM]: Add log for debug mysql sink

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update key monitor metrics in both TiCDC document and official document

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: cdc/sink/mysql.go [hunk 1]
```diff
@@ -915,6 +917,9 @@ func (s *mysqlSink) execDMLWithMaxRetries(
 					}
 				}
 				if err = tx.Commit(); err != nil {
+					if strings.Contains(err.Error(), "Error 9007: Write conflict") {
+						log.Info("write conflict happened", zap.Strings("query", dmls.sqls), zap.Any("args", dmls.values), zap.Int("bucket", bucket))
```
Comment (amyangfei) [inline cdc/sink/mysql.go:921]: We should pay attention to partition table id, but seems we can use row values to find data belongs to which partition.

Comment (3AceShowHand) [reply cdc/sink/mysql.go:921]: At the moment, `RowChangedEvent` does not record Partition-ID. maybe we should support this field

Comment (3AceShowHand) [reply cdc/sink/mysql.go:921]: For this case, the row belongs to which partition can be distinguished by row values.

Comment (3AceShowHand) [reply cdc/sink/mysql.go:921]: For this case, the row belongs to which partition can be distinguished by row values.

File: cdc/sink/mysql.go [hunk 2]
```diff
@@ -897,7 +899,7 @@ func (s *mysqlSink) execDMLWithMaxRetries(
 				}
 				for i, query := range dmls.sqls {
 					args := dmls.values[i]
-					log.Debug("exec row", zap.String("sql", query), zap.Any("args", args))
+					log.Info("exec row", zap.String("sql", query), zap.Any("args", args), zap.Int("bucket", bucket))
```
[the remark this answers is not in the document]
Comment (3AceShowHand) [reply cdc/sink/mysql.go:902]: yes, it would be many logs, but it may give us some useful information.

[the remark this answers is not in the document]
Comment (3AceShowHand) [reply cdc/sink/mysql.go:902]: I think we can convert this to `Debug` level, and when the problem occurred, change log-level to debug.

**Pull request closed without merging**